R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -10,6 +10,23 @@ vignette: >
   \usepackage[utf8]{inputenc}
 ---
 
+<!--
```

Comment:
I don't think we need license headers in doc files -- I don't think others have this.
You could add one to `DESCRIPTION` for good measure but it's really conf.